lib/elements/elements.dart

```diff
@@ -111,6 +111,10 @@ class DButton extends DElement {
   set disabled(bool value) {
     buttonElement.disabled = value;
   }
+
+  set textColor(String color) {
+    buttonElement.style.color = 'white';
```

Comment:
Should this be `= color`?

Reply:
Ahh yes, I kept it white for testing and then forgot to make it as color. My bad. I have made the required changes. Please have a look